third_party/pkcs11/pkcs11t.h

```diff
@@ -53,11 +53,11 @@ typedef CK_BYTE           CK_UTF8CHAR;
 typedef CK_BYTE           CK_BBOOL;
 
 /* an unsigned value, at least 32 bits long */
```

Comment:
This file is currently a pristine copy of the official PKCS#11 header file; if we're going to deviate from that it would be good to have a comment marking the change.  Maybe something like
```
/* Local modification: the CK_LONG/CK_ULONG types are mapped to local typedefs
 * to allow the tests to run against PKCS#11 libraries that use a CK_[U]LONG type 
 * that is a different size from `long int`.
 */
```